include/openthread/commissioner.h

```diff
@@ -119,6 +119,19 @@ OTAPI otError OTCALL otCommissionerAddJoiner(otInstance *        aInstance,
  */
 OTAPI otError OTCALL otCommissionerRemoveJoiner(otInstance *aInstance, const otExtAddress *aEui64);
 
+/**
+ * This function gets the Provisioning URL.
+ *
+ * @param[in]    aInstance       A pointer to an OpenThread instance.
+ * @param[out]   aLength         A pointer to `uint16_t` to return the length (number of chars) in the URL string.
```

Comment:
Can this be NULL?

Reply:
Technically should not be `NULL`, but we allow it. If it's `NULL` function also returns `NULL` (to indicate bad argument). Will update the doc to mention this.